profiles/api.py

```diff
@@ -256,3 +285,125 @@ def is_user_info_complete(user):
         and hasattr(user, "legal_address")
         and user.legal_address.is_complete
     )
+
+
+def import_alums(alums, header):
+    """
+    Import alums
+    """
+    try:
+        user = User.objects.get(email=alums.learner_email)
+    except User.DoesNotExist:
+        user = User.objects.create(
+            email=alums.learner_email, username=alums.learner_email
+        )
+
+        from profiles.models import Profile, LegalAddress
+
+        LegalAddress.objects.create(user=user)
+        Profile.objects.create(user=user)
+        user.username = usernameify("", user.email)
+        user.save()
+
+        log.info(
+            "User does not exist against that email=%s but created without password and incomplete profile",
+            alums.learner_email,
+        )
+    bootcamp_start_date = datetime.strptime(alums.bootcamp_start_date, "%Y-%m-%d %H:%M")
+    bootcamp_end_date = datetime.strptime(alums.bootcamp_end_date, "%Y-%m-%d %H:%M")
+    if is_naive(bootcamp_start_date):
+        bootcamp_start_date = make_aware(bootcamp_start_date)
+        bootcamp_end_date = make_aware(bootcamp_end_date)
+
+    bootcamp_run = BootcampRun.objects.get(
+        title=alums.bootcamp_run_title,
+        bootcamp__title=alums.bootcamp_name,
+        start_date__gte=bootcamp_start_date - timedelta(days=1),
+        start_date__lte=bootcamp_start_date + timedelta(days=1),
+        end_date__gte=bootcamp_end_date - timedelta(days=1),
+        end_date__lte=bootcamp_end_date + timedelta(days=1),
+    )
+
+    enrollment, created = BootcampRunEnrollment.objects.get_or_create(
+        user=user, bootcamp_run=bootcamp_run
+    )
+    if created:
+        enrollment.user_certificate_is_blocked = True
+        enrollment.save()
+        log.info(
+            "User=%s successfully enrolled in bootcamp run %s",
+            alums.learner_email,
+            alums.bootcamp_run_title,
+        )
+    else:
+        log.info(
+            "User=%s already exists in bootcamp run %s",
+            alums.learner_email,
+            alums.bootcamp_run_title,
+        )
+
+
+def parse_alumni_csv(csv_path):
+    """
+    Read CSV file and convert to Alumns object
+
+    Args:
+        csv_path(str): Path to the CSV file
+
+    Returns:
+        (list of Alumns, list):
+    """
+    with open(csv_path) as csv_file:
+        rows = list(csv.reader(csv_file))
```

Comment:
I think you can use `csv.DictReader` here. That will save you the logic of storing the header row and looking up indexes to columns there.